SysTableParse.py

```diff
@@ -123,144 +154,194 @@ def cardtotex(card):
 			r_b=float(relative_b)
 			r_s=float(relative_s)
 			sysname = vals[0]
-			sysname = sysname.replace('NORM',' Normalization')
-			if 'SHAPE' in sysname:
-				sysname = sysname.replace('SHAPE','') + ' Shape'
+			#sysname = sysname.replace('NORM',' Normalization')
```

Comment:
Why did you take this out?

Reply:
Just because we had discussed using the names directly from the datacards in the list of systematics. Thought I'd keep it consistent.